service/resource/servicev2/create_test.go

```diff
@@ -0,0 +1,108 @@
+package servicev2
+
+import (
+	"context"
+	"testing"
+
+	"github.com/giantswarm/apiextensions/pkg/apis/provider/v1alpha1"
+	"github.com/giantswarm/micrologger/microloggertest"
+	apismetav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes/fake"
+	apiv1 "k8s.io/client-go/pkg/api/v1"
+)
+
+func Test_Resource_Service_newCreateChange(t *testing.T) {
+	testCases := []struct {
+		Obj             interface{}
```

Comment:
Could we have these struct members as private? A description field here would be nice too, aligning with `desired_test`

Reply:
Good point, fixed. These were copy paste from another resource.